### ISSUE TITLE
Clarification for using the datasets

### ISSUE THREAD

**Issue by geniusqwerty12** [Max-1234-hub/light-VGG11#3]
Hi, can you provide either a step-by-step or clear instructions on how to use the .pkl datasets provided in the link. I would like to try this on my local computer. A response is very appreciated.

**Comment by Max-1234-hub** [comment]
Sorry for the late reply.
The file "Used_AudioAndLabels.pkl" is the original dataset, which contains 5336 samples, i.e., 3363 distress calls and 1973 natural barn sounds. Before inputting them into the neural network, you should process the dataset first, as stated in the first paragraph of Section 2.3 Automatic Classification. Here, I repeat the statements: "First, recordings in the audio files were transformed to log-Mel spectrograms, then image-wise normalized to the range [0, 1]. After that, we split these normalized spectrograms into training, validation and test sets based on a fivefold cross-validation technique...". As for how to get the log-Mel spectrograms, please see the detailed process in Section 2.3.1 Input preparation. I will give more detailed instructions for training in the readme file. Thanks for your attention.
